fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/ScopedMetricHavingDataServletSpec.groovy

```diff
@@ -0,0 +1,99 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web.endpoints
+
+class ScopedMetricHavingDataServletSpec extends BaseDataServletComponentSpec {
```

Comment:
I'm little curious, how does this Spec endup making the changes to `ExpectedTablesFullViewEndpointSpec.groovy` Spec above?

Reply:
I needed to add a scoped metric to our metric configuration in order to reproduce the bug. The `ExpectedTablesFullViewEndpointSpec` prints all the metrics for each table. So I needed to update the shapes table's output with the new metric.